### ISSUE TITLE
Server status: cannot connect

### ISSUE THREAD

**Issue by jflorian** [smblott-github/text-aid-too#12]
New user here on Fedora 28 with chromium-66.0.3359.181-2.fc28.x86_64.  My server looks good:
~~~
$ systemctl --user status text-aid-too.service
● text-aid-too.service - Text-Aid-Too
   Loaded: loaded (/home/00/d13677/.config/systemd/user/text-aid-too.service; enabled; vendor preset: enabled)
   Active: active (running) since Wed 2018-06-06 14:40:44 EDT; 2s ago
 Main PID: 11164 (coffee)
   CGroup: /user.slice/user-10325.slice/user@10325.service/text-aid-too.service
           └─11164 /usr/bin/node /usr/bin/coffee /usr/bin/text-aid-too

Jun 06 14:40:44 mdct-dev12.dartcontainer.com systemd[1999]: Started Text-Aid-Too.
Jun 06 14:40:44 mdct-dev12.dartcontainer.com text-aid-too[11164]: server  ws://localhost:9293
Jun 06 14:40:44 mdct-dev12.dartcontainer.com text-aid-too[11164]: secret  YouBetterButterBothBuns
Jun 06 14:40:44 mdct-dev12.dartcontainer.com text-aid-too[11164]: editor  gvim -f
Jun 06 14:40:44 mdct-dev12.dartcontainer.com text-aid-too[11164]: version 1.1.6
~~~

Nonetheless, my Text-Aid-Too Options page still shows that the extension cannot connect.  I've tried with and without a secret.  I've confirmed the server is indeed listening as I can `telnet 127.0.0.1 9293`.

What next?

**Comment by smblott-github** [comment]
Can you telnet to the server port?

The default port is 7442, did you change that somehow?

**Comment by jflorian** [comment]
I was/am able to telnet and connect to port 9293.  I did not change the port on client or server, both came up as 9293 for a default.  I'm utterly confused that you're now saying the default is 7442 because... well, see the highlighted section below:

![text-aid-too](https://user-images.githubusercontent.com/101996/41101939-939dc108-6a33-11e8-9744-723e729c08e7.png)

... and the README here indicates 9293 as well.

**Comment by jflorian** [comment]
Just revisiting this because my need for an extension like this was the one barrier preventing me from adopting chromium completely.  I installed the Surfing Keys extension because it claimed an ability to edit text areas with Vim.  Sure enough it did.  Wait!  Why does Vim's titlebar say text-add-too is in the filename?!?!?  So, I played a bit more with Surfing Keys, but I'm too used to Vimium so I uninstalled it... eager to get back to what appeared to be a working Text-Aid-Too.

Long story short, I have no idea what was preventing it from working.  All of my debugging said it should, but it wouldn't yet now it does.  I'll take mysterious function over logical dysfunction and be happy at this point.

**Comment by smblott-github** [comment]
(Just back from a long trip.)

Glad to see it's working!
